recipe/meta.yaml

```diff
@@ -52,7 +52,7 @@ requirements:
     - boost 1.61.*  # [not win]
     - blas 1.1 {{ variant }}  # [not win]
     - openblas 0.2.18|0.2.18.*  # [not win]
-    - libxml2  # [not win]
+    - libxml2 2.9.3  # [not win]
```

Comment:
There is a bug in 2.9.4. Not sure how `openturns` uses libxml2 but this is desirable in most cases.